source/common/runtime/runtime_features.cc

```diff
@@ -112,7 +112,8 @@ constexpr const char* disabled_runtime_features[] = {
     "envoy.reloadable_features.test_feature_false",
     // gRPC Timeout header is missing (#13580)
     "envoy.reloadable_features.ext_authz_measure_timeout_on_check_created",
-
+    // The cluster which can't extract secret entity by SDS to be warming and never activate.
```

Comment:
I can't parse this. Can you describe to me what this does and I can help with some better language?